projects/igniteui-angular/src/lib/grids/cell.component.ts

```diff
@@ -813,7 +813,9 @@ export class IgxGridCellComponent implements OnInit, OnChanges, OnDestroy {
      * @internal
      */
     public pointerdown = (event: PointerEvent) => {
-        event.preventDefault();
+        if(this.platformUtil.isFirefox) {
+            event.preventDefault();
+        }
```

Comment:
Can you consider moving this check below the check for cell selection mode because we break the default behavior and the user cannot make a text selection? We need this logic only in case we have range selection enabled.